infra/module-swapper/cmd/testdata/example-with-steps-submodules/examples/example-one/main.tf

```diff
@@ -0,0 +1,27 @@
+module "test-module" {
+  source  = "terraform-google-modules/example-with-steps-submodules/google"
+  version = "~> 3.2.0"
+
+  project_id   = var.project_id # Replace this with your project ID in quotes
```

Comment:
Based on the comment `# Replace this with your project ID in quotes` I'm wondering if the best approach would be to hardcode the _project_id_ on the main.tf or keep the it as it is (variables) and add the value into a `terraform.tfvars` file.

Reply:
Thanks @renato-rudnicki - This is testdata, and this particular field isn't actually critical to the test.